config/locales/ru/map.yml

```diff
@@ -59,17 +59,17 @@ ru:
   javascript:
     recurrence:
       day: "Ежедневно"
-      monday: "По Понедельникам"
-      tuesday: "По Вторникам"
-      wednesday: "По Средам"
-      thursday: "По Четвергам"
-      friday: "По Пятницам"
-      saturday: "По Субботам"
-      sunday: "По Воскресеньям"
+      monday: "По понедельникам"
+      tuesday: "По вторникам"
+      wednesday: "По средам"
+      thursday: "По четвергам"
+      friday: "По пятницам"
+      saturday: "По субботам"
+      sunday: "По воскресеньям"
     timing_labels:
-      course_fallback: "Это еженедельный курс"
+      course_fallback: "Это еженедельный курс" # ???
```

Comment:
Fallback means the label that get's used when none of the others apply. It's not the best label, I should probably change it to "weekly:"